src/modules/accommodations/accommodations.controller.ts

```diff
@@ -23,4 +43,36 @@ export class AccommodationsController {
       limit: searchQueryParams.limit,
     };
   }
+
+  @Post()
+  @CreateAccommodationSwaggerDecorator()
+  async createAccommodation(@Body() createDto: CreateAccommodationDto) {
```

Comment:
your route is not protected. you added a new migration to add an owner column to the accommodation entity but you are not assigning any value to the owner when you are creating accommodation.